### PR TITLE
Adds  __set__ to the reagent_label Descriptor to allow updates

### PR DESCRIPTION
**Pull request by PerLundmark**
Adds a  __set__ implementation to the Descriptor handling reagent label lists. Allows setting the reagent label (index) of artifacts.

### CODE REVIEW

File: genologics/descriptors.py [hunk 1]
```diff
@@ -542,6 +542,20 @@ def __get__(self, instance, cls):
                 pass
         return self.value
 
+    def __set__(self, instance, values):
+        instance.get()
+        nodes = instance.root.findall('reagent-label')
+        if not nodes:
+            root = instance.root
+            for value in values:
+                rl_node = ElementTree.SubElement(root, 'reagent-label', name=value)
```
Comment (GitEdvard) [inline genologics/descriptors.py]: rl_node is not used anywhere else. Is it a mistake? If it has to be there, I think 'dummy' is better.

**Review by GitEdvard — COMMENTED** [review]
OK, I have 1 comment. Tell me when you want me to merge.

File: genologics/descriptors.py (hunk 1 above)
Comment (PerLundmark) [reply genologics/descriptors.py]: Can run without it, I'll nuke it

Comment (PerLundmark) [reply genologics/descriptors.py]: Done

**Pull request merged**